tests/python_package_test/test_sklearn.py

```diff
@@ -719,6 +719,21 @@ def test_predict():
     with pytest.raises(AssertionError):
         np.testing.assert_allclose(res_engine, res_sklearn_params)
 
+    # Test multiclass binary classification
+    num_samples, num_classes = 20, 2
+    rng = np.random.Generator(np.random.PCG64())
+    X_train = rng.uniform(low=0, high=1, size=[num_samples, 3])
+    y_train = rng.choice([0, 1], size=num_samples)
```

Comment:
The randomness here could sometimes lead to the test failing. For example, it's possible for `y_train` to contain all 0s, which would result in an error at training time... not too unlikely, given the small data sizes involved here.

I directly experienced that trying to run the reproducible example from #6519.

Could you please remove randomness here and just create these arrays with `np.array` literals or something else deterministic?

Reply:
I remove randomness in https://github.com/microsoft/LightGBM/pull/6524/commits/66b372ae5d5dcdedbf9862fa877d2e65af19382b.